src/components/HURUmap/Tutorial/TutorialItem/index.js

```diff
@@ -0,0 +1,93 @@
+import RichTypography from "@commons-ui/core/RichTypography";
```

Comment:
Since we're using `TourProvider`, shouldn't we call this `TourStep` or `TutorialStep` if we don't want to use `Tour`?